mert/SerScorer.cpp

```diff
@@ -69,19 +69,19 @@ void SerScorer::prepareStats(size_t sid, const string& text, ScoreStats& entry)
       }
       if (errors == 0) {
         n_ser = 0;
-	break;
+        break;
       }
     }
   }
   ostringstream stats;
-  stats << n_ser;
+  stats << n_ser << " " << 1; // sentence error (0 or 1), number of sentences (1)
   string stats_str = stats.str();
   entry.set(stats_str);
 }
 
 float SerScorer::calculateScore(const vector<ScoreStatsType>& comps) const
 {
-  return comps[0] / float(m_ref_sentences[0].size());
+  return comps[0] / float(comps[1]);
```

Comment:
In case there are 4 references and none of the references match the hypothesis then this might return the SER as 4. Is this intentional?

Reply:
no, this will always be 1 because the counting happens outside the "rid" loop above (line 57) which iterates over multiple references

Reply:
👍